compiler-rt/lib/hwasan/hwasan_platform_interceptors.h

```diff
@@ -66,7 +66,7 @@
 #define SANITIZER_INTERCEPT_MEMCPY 0
 
 #undef SANITIZER_INTERCEPT_MEMCMP
-#define SANITIZER_INTERCEPT_MEMCMP 0
+#define SANITIZER_INTERCEPT_MEMCMP 1
```

Comment:
Please note the comments at the top of the file:
- don't forcibly #define as 1 because it prevents interception being defined elsewhere as conditional (e.g., Fuchsia really doesn't like interceptors)
- corollary: comment out the #define as 0 (instead of deleting it) as a placeholder

Reply:
and also comment out the undef

Reply:
Done.